### ISSUE TITLE
OCCTAX - Avertissement sur le bouton annuler

### ISSUE THREAD

**Issue by jbrieuclp** [PnX-SI/GeoNature#686]
En lien avec #338 : émettre un avertissement lors du clic sur le bouton annuler au niveau de la saisie. "Etes-vous certain de vouloir annuler". Plusieurs se sont fait avoir en ratant le bouton "Enregistrer" juste à côté.

**Comment by RomainBaghi** [comment]
Oui, pareil de mon coté. Et même lorsque que l'on quitte la page sans le vouloir, en lien avec ce ticket : https://github.com/PnX-SI/GeoNature/issues/578
